intercept/src/intercept.cpp

```diff
@@ -2768,7 +2768,7 @@ void CLIntercept::combineProgramStrings(
         if( ( lengths == NULL ) ||
             ( lengths[i] == 0 ) )
         {
-            if( strings[i] != NULL )
+            if( *&strings != NULL && strings[i] != NULL )
```

Comment:
Can this just check for `strings != NULL` instead of `*&strings != NULL`?

Reply:
Urgh, sorry had head in wrong gear it seems.